test/with_dummyserver/test_socketlevel.py

```diff
@@ -1387,6 +1389,14 @@ def socket_handler(listener):
                         pool.request("GET", "/", timeout=SHORT_TIMEOUT)
                     context.load_default_certs.assert_not_called()
 
+    def test_load_verify_locations_exception(self):
+        """
+        Ensure that load_verify_locations raises SSLError for all backends
+        """
+        with pytest.raises(SSLError) as exc:
+            ssl_wrap_socket(None, ca_certs=os.devnull)
+        assert exc.type == SSLError
```

Comment:
Isn't this assert redundant with the `pytest.raises()`?

Reply:
Nice catch, thanks! Fixed.